dolphin/resource_manager/storages.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2013 NetApp
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""The shares api."""
+
+import copy
+
+from os_service_types import exc
+from oslo_log import log
+from oslo_utils import uuidutils, timeutils
+
+from dolphin import db
+from dolphin.db.sqlalchemy import api as db
+from dolphin.driver_manager import manager as drivermanager
+
+LOG = log.getLogger(__name__)
+
+
+def device_model(body):
+    device = DeviceModel()
+    device.id = uuidutils.generate_uuid()
+    device.created_at = timeutils.utcnow()
+    if body.get('name'):
+        device.name = body.get('name')
+    if body.get('description'):
+        device.description = body.get('description')
+    if body.get('vendor'):
+        device.vendor = body.get('vendor')
+    if body.get('status'):
+        device.status = body.get('status')
+    if body.get('total_capacity'):
+        device.total_capacity = body.get('total_capacity')
+    if body.get('manufacturer'):
+        device.manufacturer = body.get('manufacturer')
+    if body.get('model'):
+        device.model = body.get('model')
+    if body.get('firmware_version'):
+        device.firmware_version = body.get('firmware_version')
+    return device
+
+
+class DeviceModel:
+    def __init__(self):
+        self.id = None
```

Comment:
Follow python style of initialization self.id=self.name \
                                                        self.description = None       
:) :)